docs/quickstart.md

```diff
@@ -0,0 +1,176 @@
+# Initial Setup / Operations
+
+`SSO` is used to provide single-sign-on authentication and authorization for internal web
+applications, however, there are a few one-time prerequisites that need to be handled manually.
+
+**NOTE: all commands should be performed on your HOST OS.**
+
+## Google OAuth Setup
+
+Configuring `sso` requires that we first configure a Google OAuth Project.
+
+#### Create a new Google Cloud project
+
+Use a web browser to access the Google Cloud [Resource Manager](
+https://console.developers.google.com/cloud-resource-manager). Log into your account, if you have
+not already done so.
+
+![Manage Resources](images/setup-manage_resources.jpg)
+
+Select "Create Project". Provide the requested details (Project Name, etc), and click "Create". You
+will have a notification when your project has finished being created; this should only take a
+few moments.
+
+![Notification](images/setup-notification.jpg)
+
+#### Configure Google OAuth
+
+Once your project has been created, click [here](
+https://console.cloud.google.com/apis/credentials/consent) to configure the "OAuth consent screen"
+for your project. Make sure that the name of the project you have created appears next to the words
+"Google Cloud Platform" in the toolbar at the top of the website; if it does not, switch to your
+project by clicking the name that appears there, and selecting the project you just created.
+
+Fill in the field "Product name shown to users". We recommend the name `SSO Authenticator` for this
+field. Hit "Save" after filling in this field.
+
+![OAuth Consent Screen](images/setup-consent_screen.jpg)
+
+After this, click [here](https://console.cloud.google.com/apis/credentials) to configure
+credentials for the service. Click the button labeled "Create credentials", and select the option
+"OAuth client ID".
+
+![Create Credentials Dropdown](images/setup-create_credentials.jpg)
+
+The next screen will ask you to choose an "Application Type"; select "Web application". You will be
+asked to fill in three fields.
+- **Name**: Any appropriate name is fine (e.g. `Dev`).
+- **Authorized JavaScript origins**: We recommend leaving this field blank.
+- **Authorized redirect URIs**: Add the URI of your `sso-proxy` deployment, with the path suffix
```

Comment:
I'm pretty sure this needs to point at the _sso-auth_ deployment, not the sso-proxy deployment.

Reply:
correct, the location of the quickstart is `http://sso-auth.sso-test.io/oauth2/callback`, fixing.